ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -116,6 +116,13 @@
   @Default("1*1024*1024")
   public final int storeCompactionOperationsBytesPerSec;
 
+  /**
+   * The check interval used in compaction rate throttler.
+   */
+  @Config("store.compaction.throttler.check.interval.ms")
+  @Default("-1")
```

Comment:
Please add some comment describing what does -1 mean. Looks like if it's <0,  rate is checked on every call.

Reply:
yes. You are right. Added in the comment.